Quoteau/Quoteau/TabBarVC.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  TabBarVC.swift
+//  Quoteau
+//
+//  Created by Wiktor Górka on 17/12/2019.
+//  Copyright © 2019 Lunar Logic. All rights reserved.
+//
+
+import UIKit
+
+class TabBarVC: UITabBarController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setupViewControllers()
+    }
+    
+    
+    fileprivate func setupViewControllers() {
+        
+        let homeViewController = templateNavController(unselectedImage: UIImage.init(systemName: "house")!, selectedImage: UIImage.init(systemName: "house.fill")!)
+        
+        let cameraViewController = templateNavController(unselectedImage: UIImage.init(systemName: "plus")!, selectedImage: UIImage.init(systemName: "plus")!)
+        
+        let profileViewController = templateNavController(unselectedImage: UIImage(systemName: "person")!, selectedImage: UIImage(systemName: "person.fill")!)
+        
+        viewControllers = [homeViewController, cameraViewController, profileViewController]
+        
+        guard let items = tabBar.items else { return }
+        
+        for item in items {
+            item.imageInsets = UIEdgeInsets(top: 4, left: 0, bottom: -4, right: 0)
```

Comment:
I'd consider building an abstraction on top of these numbers, if it makes sense. Do they represent something specific? Can they be named? Is there an option they will be reused?